src/components/App.module.css

```diff
@@ -1,3 +1,7 @@
+.main {
+    font-family: 'Open Sans Condensed', sans-serif;
```

Comment:
You were right to define this at `body`. Unless you don't want it to be inherited everywhere?